### PR TITLE
GF-46 - Redux implemetetion for constructors

### PR DESCRIPTION
**Pull request by MIKALAKE**

### CODE REVIEW

File: src/show/pages/private/Admin/index.jsx [hunk 1]
```diff
@@ -55,40 +58,35 @@ const Admin = () => {
...
-        setCreateConstructorModal(false);
-      });
-  };
 
   const postEvent = e => {
     e.preventDefault();
     dispatch(addEventAsync(event));
     setCreateEventModal(false);
   };
+  const postConstructor = e => {
```
Comment (iamalextodoran) [inline src/show/pages/private/Admin/index.jsx:79]: Empty line before this.

File: src/show/components/ConstructorCardAdmin/index.jsx [hunk 2]
```diff
@@ -1,41 +1,27 @@
...
+} from '../../../redux/constructorsSlice';
 import { Button, ConstructorInfoCard, Modal, TextField } from '../';
 
-const ConstructorCardAdmin = ({
-  constructor,
-  constructors,
-  setConstructors
-}) => {
+const ConstructorCardAdmin = ({ constructor }) => {
   const [editConstructorModal, setEditConstructorModal] = useState(false);
```
Comment (iamalextodoran) [inline src/show/components/ConstructorCardAdmin/index.jsx]: Move this bellow editedConstructor

File: src/redux/constructorsSlice.js [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+      return { ...state, constructors: newConstructors };
+    }
+  }
+});
+
+export const {
+  deleteConstructor,
+  editConstructor,
+  getConstructor,
+  getConstructors
```
Comment (iamalextodoran) [inline src/redux/constructorsSlice.js]: What are those?

**Review by iamalextodoran — CHANGES_REQUESTED** [review]

File: src/redux/constructorsSlice.js (hunk 3 above)
Comment (MIKALAKE) [reply src/redux/constructorsSlice.js]: idk man, i saw tutorial i do tutorial

File: src/show/pages/private/Admin/index.jsx (hunk 1 above)
Comment (MIKALAKE) [reply src/show/pages/private/Admin/index.jsx:79]: done

File: src/index.js [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
-import store from './redux/store';
 import { Provider } from 'react-redux';
 import ReactDOM from 'react-dom/client';
+import store from './process/redux/store';
```
Comment (iamalextodoran) [inline src/index.js]: Group with Application

**Review by iamalextodoran — CHANGES_REQUESTED** [review]

**Review by iamalextodoran — APPROVED** [review]

**Pull request merged**